docs/pages/Configuration.md

```diff
@@ -87,8 +87,9 @@ Some general information:
         + `<virtual path>: <physical path>` (default `/: ./`): Any virtual path mapping can be defined here.  
           *(Only available for projects of type `module`)*  
           It is recommended that modules include their namespace in the virtual path and use the `/resources` prefix (e.g. `/resources/my/first/library/module-xy`).
-- `cachebuster`:
-    - `signatureType`: `time` or `hash`. By default, the generated cachebuster info file signatures are based on timestamps (`time`). In setups like CI environments, a mechanism based on file hashes (`hash`) might be more reliable.
+- `propertiesFileEncoding`:  
```

Comment:
- moved to "resources.configuration"
- rename according to impl.

Reply:
Done